tests/unit/tests_high_five_base.cpp

```diff
@@ -247,8 +247,8 @@ BOOST_AUTO_TEST_CASE(HighFiveGroupAndDataSet) {
 BOOST_AUTO_TEST_CASE(HighFiveExtensibleDataSet) {
     const std::string FILE_NAME("create_extensible_dataset_example.h5");
     const std::string DATASET_NAME("dset");
-    constexpr double t1[3][1] = {{2.0}, {2.0}, {4.0}};
-    constexpr double t2[1][3] = {{4.0, 8.0, 6.0}};
+    constexpr long double t1[3][1] = {{2.0}, {2.0}, {4.0}};
```

Comment:
Seems fine if, although optimally we'd test all data type mappings, just in case "something" breaks?

Reply:
Yes, I was surprise a basic all-types test is missing